torch/quantization/fake_quantize.py

```diff
@@ -65,9 +63,26 @@ def extra_repr(self):
             self.fake_quant_enabled, self.observer_enabled,
             self.scale, self.zero_point)
 
-default_fake_quant = FakeQuantize
+    def _save_to_state_dict(self, destination, prefix, keep_vars):
+        super(FakeQuantize, self)._save_to_state_dict(destination, prefix, keep_vars)
+        destination[prefix + 'quant_min'] = self.quant_min
+        destination[prefix + 'quant_max'] = self.quant_max
```

Comment:
aren't quant_min/quant_max also parameters of the model? i.e. they are not learnable and one would need to pass them to constructor in code before even loading the statedict?

Reply:
Yes, going by our discussion earlier, these are config params, not learnable params. Will remove them from state dict